backend/Makefile

```
@@ -280,20 +284,51 @@ test-integration-serial: composition ## Escape hatch: the old sequential lane, a
 	fi; \
 	rm -f "$$log"; echo "OK: integration passed with 0 skips"
 
-# The integration lane above runs the same harness on the SMB tier as a
-# standing canary; this target is the tier the PERF-7 SLO binds at.
+# PERF-3/PERF-7 at the tier the SLO binds at, and the only target that WRITES a
+# record. The suite used to ride the standing integration lane at the SMB tier
+# as well; it does not any more (see its own file comment), so this target and
+# bench-perf-check below are the two ways it runs.
 bench-perf: composition ## PERF-3/PERF-7 benchmark harness on the mid-market §6.7 tier (needs db-up; seeds 250k contacts)
```

checkmake (0.3.2)

[warning] Target body for "bench-perf" exceeds allowed length of 5 lines (6).

(maxbodylength)